packages/did-eth-registry/Dockerfile

```diff
@@ -1,56 +1,17 @@
-FROM debian:stable-slim as python-builder
+FROM ghcr.io/xmtp/foundry:latest
 
-# python3.10 is required for node-gyp
```

Comment:
Python 3.10 is no longer required on the latest hardhat and node versions (solc 0.8.21)